django_filters/filters.py

```diff
@@ -253,10 +253,13 @@ def filter(self, qs, value):
         return qs.distinct() if self.distinct else qs
 
     def get_filter_predicate(self, v):
+        name = self.field_name
+        if name and self.lookup_expr:
```

Comment:
`self.lookup_expr` is [`'exact'` by default](https://github.com/carltongibson/django-filter/blob/ab31272025a78c77330ffa105456bc0a522fd92d/django_filters/filters.py#L69). It's always _truthy_

Do we want `if name and self.lookup_expr != 'exact':...`?

Reply:
Ah, that was from before I discovered (by way of tests) that this filter didn't use `__exact`, hence the superflous conditional. I opted to keep the `__exact` since pretty much all other filters default to `__exact`, but you're right that we might as well leave it off, too.